core/src/autogluon/core/_setup_utils.py

```diff
@@ -18,7 +18,7 @@
     'pandas': '>=1.0.0,<2.0',
     'scikit-learn': '>=0.23.2,<0.25',  # 0.22 crashes during efficient OOB in Tabular
     'scipy': '>=1.5.4,<1.7',
-    'gluoncv': '>=0.10.2.post0,<0.10.4',
+    'gluoncv': '==0.11.0b20210611', # '>=0.10.2.post0,<0.10.4',
```

Comment:
We can change this after GluonCV is upgraded.

Reply:
You can now use gluoncv>=0.10.3,<0.10.4